introduction.md

```diff
@@ -0,0 +1,2 @@
+This is file is to learn how to fork from Deadly Vipers dojo_rules repository, clone it to my local, then add, commit and
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.